File: src/AutoRest.SdkExplorer/Model/Azure/AzureResourceGroup.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

#nullable disable

namespace AutoRest.SdkExplorer.Model.Azure
{
    public class AzureResourceGroup
    {
        public string Name { get; set; }
        public string Id { get; set; }
        public string Region { get; set; }

        public AzureResourceGroup()
        {
        }

        public AzureResourceGroup(string name, string id, string region)
        {
            this.Name = name;
            this.Id = id;
            this.Region = region;
        }
    }
}

File: src/AutoRest.SdkExplorer.TypeScript/src/Model/Azure/AzureResourceGroup.ts
export class AzureResourceGroup implements AutoRest.SdkExplorer.Interface.AzureResourceGroup {
    name: string;
    id: string;
    region: string;

    constructor(data: AutoRest.SdkExplorer.Interface.AzureResourceGroup) {
        this.name = data.name!;
        this.id = data.id!;
        this.region = data.region!;
    }
}

